### PR TITLE
Add components and styling

### PR DESCRIPTION
**Pull request by David-Lanzz**
- I have added a component for the preview of the books 
- I have added a component for adding books to the dom 
- Basic styling 
- I have installed react router

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @David-Lanzz,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a descriptive readme document. Kindly use the template to update your readme.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/reportWebVitals.js [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```
Comment (VICTOR-CHUKWU) [inline src/reportWebVitals.js:15]: - [ ] [Optional]remove this file as it is part of the boilerplate created from create react app

File: src/components/books.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { NavLink } from 'react-router-dom';
+import '../styles/books.css';
+import PropTypes from 'prop-types';
+/* eslint-disable no-unused-vars */
```
Comment (VICTOR-CHUKWU) [inline src/components/books.jsx:4]: - [ ] [OPTIONAL] instead of disabling linters kindly remove `modifybooks` as it is not used

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @David-Lanzz , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**